src/applications/disability-benefits/all-claims/config/form0781/index.js

```diff
@@ -13,4 +19,18 @@ export const form0781PagesConfig = {
     uiSchema: workflowChoicePage.uiSchema,
     schema: workflowChoicePage.schema,
   },
+  manualUploadPage: {
+    path:
+      'disability/file-disability-claim-form-21-526ez/additional-forms/mental-health-statement/upload',
+    uiSchema: manualUploadPage.uiSchema,
+    depends: formData => showManualUpload0781Page(formData),
+    schema: manualUploadPage.schema,
+  },
```

Comment:
This looks like it's referring to [[3.3], the Upload 0781 page](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=9442-62961&t=OwUuVSYswnhT7tYQ-4), when I thought the corresponding page for [#97067](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97067) was [[3.2], the 0781 choice page](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=9442-72041&t=OwUuVSYswnhT7tYQ-4).

Reply:
Oh, never mind. I realize now that the `workflowChoicePage` is [[3.2], the 0781 choice page](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=9442-72041&t=OwUuVSYswnhT7tYQ-4).